src/ui/qgsdatadefinedsizelegendwidget.ui

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
- <class>QgsDataDefinedSizeLegendDialog</class>
- <widget class="QDialog" name="QgsDataDefinedSizeLegendDialog">
+ <class>QgsDataDefinedSizeLegendWidget</class>
+ <widget class="QWidget" name="QgsDataDefinedSizeLegendWidget">
```

Comment:
should this be QgsPanelWidget? not sure if it makes any ultimate difference though.